entrapment.sh

```diff
@@ -0,0 +1,11 @@
+_OLD_LD_LIBRARY_PATH=$LD_LIBRARY_PATH
+function ld_library_path() {
+    if [ -n "$_OLD_LD_LIBRARY_PATH" -a "$_OLD_LD_LIBRARY_PATH" != "$LD_LIBRARY_PATH" ]
+    then
+	LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$_OLD_LD_LIBRARY_PATH
```

Comment:
whitespace!

Reply:
I really need to teach my Emacs to use spaces since I always manually use them anyway...